pilot/pkg/networking/core/v1alpha3/listener.go

```diff
@@ -1290,22 +1284,20 @@ func (configgen *ConfigGeneratorImpl) buildSidecarOutboundThriftListenerOptsForP
 	}}
 }
 
-func (configgen *ConfigGeneratorImpl) buildSidecarOutboundTCPListenerOptsForPortOrUDS(node *model.Proxy, destinationCIDR *string, listenerMapKey *string,
-	currentListenerEntry **outboundListenerEntry, listenerOpts *buildListenerOpts,
-	pluginParams *plugin.InputParams, listenerMap map[string]*outboundListenerEntry,
-	virtualServices []model.Config, actualWildcard string) (bool, []*filterChainOpts) {
-
-	// first identify the bind if its not set. Then construct the key
-	// used to lookup the listener in the conflict map.
-
-	// Determine the listener address if bind is empty
-	// we listen on the service VIP if and only
-	// if the address is an IP address. If its a CIDR, we listen on
-	// 0.0.0.0, and setup a filter chain match for the CIDR range.
-	// As a small optimization, CIDRs with /32 prefix will be converted
-	// into listener address so that there is a dedicated listener for this
-	// ip:port. This will reduce the impact of a listener reload
-
+// setDefaultBind first identifies the bind if its not set. Then constructs the
+// key used to lookup the listener in the conflict map.
+//
+// If bind is empty we listen on the service VIP if and only
+// if the address is an IP address. If its a CIDR, we listen on
+// 0.0.0.0, and setup a filter chain match for the CIDR range.
+// As a small optimization, CIDRs with /32 prefix will be converted
```

Comment:
I noticed this mentions `/32` but then the code itself doesn't have any special handling, e.g. I'd expect if `strings.Contains(svcListenAddress, "/")` was true there'd be a **special** check for `strings.HasSuffix(svcListenAddress, "/32")`

Reply:
Hmm. That comment may have been an old one [I remember adding code of that form or having seen code of that form a while ago. But this code went through a lot of changes over time]